ChangeLog

```diff
@@ -49,6 +49,8 @@ Features
      independently contributed again by Paul Sokolovsky.
    * Add support for key wrapping modes based on AES as defined by
      NIST SP 800-38F algorithms KW and KWP and by RFC 3394 and RFC 5649.
+   * Add support different keys sizes to CTR_CRBG. Note: using shorter keys
+     than 256 bit may decrease security.
```

Comment:
Grammar: use "bits" instead of "bit".

It'd be more natural to say "using keys shorter than" instead of "using short keys than".